spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -259,10 +273,10 @@
     it 'redirects to how to verify when a form submission error is encountered' do
       put :update, params: invalid_params
 
-      expect(flash[:error]).to be_present
+      expect(flash[:error]).not_to be_present
       expect(subject.idv_session.skip_doc_auth).to be_nil
       expect(subject.idv_session.opted_in_to_in_person_proofing).to be_nil
-      expect(response).to redirect_to(idv_how_to_verify_url)
+      expect(response).to render_template :show
```

Comment:
We now let the form handle this for us, so we no longer need to set a flash message, and we just re-render the form (show template) vs. redirecting.